components/case_macros/Cargo.toml

```diff
@@ -0,0 +1,9 @@
+[package]
+name = "case_macros"
```

Comment:
can we move it into `tikv-util` instead of an individual component?

Reply:
Can't, a proc-macro=true crate can only export proc-macros. It would be nice if we have a `util-macros` crate though.